foundations/javascript_basics/project_rock_paper_scissors.md

```diff
@@ -1,42 +1,58 @@
+1. Your game plays against the computer. Create a function `getComputerChoice` that will randomly `return` either 'Rock', 'Paper' or 'Scissors'.
+   - Create a new function `getComputerChoice`.
+   - Write the code to let `getComputerChoice` randomly `return` one of the following string values: "Rock", "Paper" or "Scissors".
+   - Test that your function returns what you expect using `console.log` or [the browser developer tools](https://www.theodinproject.com/lessons/foundations-javascript-developer-tools) before you move on to the next step.
+1. Your game plays turn by turn. Write a function named `playRound`, which takes the `playerSelection` and `computerSelection` to play a single round, and returns a winner announcement. 
+   - Create a new function called `playRound`.
+   - Define two parameters on `playRound`: `playerSelection` and `computerSelection`. Use those two parameters to pass the human and computer choices as arguments.
+   - Make your function's `playerSelection` parameter case-insensitive (so users can input "rock", "ROCK", "RocK" or any other variation).
+   - Write the code for your `playRound` function to `return` a string value representing the round winner, such as: "You Lose! Paper beats Rock".
+   - example code:
+
+   ~~~javascript
+   function playRound(playerSelection, computerSelection) {
+   // your code here!
+   }
  
-    const playerSelection = "rock";
-    const computerSelection = getComputerChoice();
-    console.log(playRound(playerSelection, computerSelection));
-    ~~~
-
-6.  Write a NEW function called `game()`. Use the previous function _inside_ of this one to play a 5 round game that keeps score and reports a winner or loser at the end.
-    *   You have not officially learned how to "loop" over code to repeat function calls... if you already know about loops from somewhere else (or if you feel like doing some more learning) feel free to use them. If not, don't worry! Just call your `playRound` function 5 times in a row. Loops are covered in the next lesson.
-    *   At this point you should be using `console.log()` to display the results of each round and the winner at the end.
-    *   Use `prompt()` to get input from the user. [Read the docs here if you need to.](https://developer.mozilla.org/en-US/docs/Web/API/Window/prompt)
-    *   Feel free to re-work your previous functions if you need to. Specifically, you might want to change the return value to something more useful.
-    *   Feel free to create more "helper" functions if you think it would be useful.
+   const playerSelection = "rock";
+   const computerSelection = getComputerChoice();
+   console.log(playRound(playerSelection, computerSelection));
+   ~~~
+
+1. Your game plays for 5 rounds. Write a function called `game` which will keep score for 5 rounds, utilizing your 'playRound' function, before it declares a winner:
```

Comment:
[Proofreading] Parts of this step were a bit unclear, so I gave some ideas on how to revise it. Check out my suggestions below:
```suggestion
1. Write a function called `game` which will keep score for 5 rounds and utilize your 'playRound' function before it declares a winner:
```

Reply:
I hope you saw my last comment. We're still finding a direction for this so no need to review just yet. Thank you however :)